clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/ops/AbstractEnableDisableAtomicOperation.groovy

```diff
@@ -76,14 +78,26 @@ abstract class AbstractEnableDisableAtomicOperation implements AtomicOperation<V
         return true
       }
 
+      //IF LB and TG is present and desiredPercentage is present -> throw error
+      //De-registering specific tasks from an ALB is currently not supported for Titu
+      if (disable && description.desiredPercentage && loadBalancingClient && job.labels.containsKey("spinnaker.targetGroups")) {
+        log.error("Could not ${verb} ServerGroup '$serverGroupName' in region $region! disabling by percentage for Server Groups with Target Groups is not supported for Titus")
```

Comment:
Would be nice to avoid the excessive long lines, perhaps something like:
```
log.error(
  "Could not {} server group '{}' in region {}! Disably by percentage for server groups ...",
  verb, serverGroupName, region
)
```